### ISSUE TITLE
Test CPL on Windows 

### ISSUE THREAD

**Issue by sayerhs** [sayerhs/cpl#5]
Ensure that all features work on Windows OS. 

- [x] Detect and load CPL config from several locations 
  - [x] `caelus.yaml` in `APPDATA`
  - [ ] `caelus.yaml` in `USERPROFILE`
  - [x] `caelus.yaml` in current working directory
  - [x] Files pointed to by `CAELUSRC_SYSTEM` and `CAELUSRC` 
- [x] Creation of `caelus-environment.cmd` 
- [x] MPICH and MSMPI settings, execution in parallel
- [x] Installation process (user docs) for `conda` and `virtualenv` (See #2)
- [x] Run tutorials to test execution in parallel and serial modes

**Comment by cnsidero** [comment]
- Upon inspecting the source code, I don't believe the second CPL config location was implemented so that wasn't checked.

- When using the CAELUSRC option CPL returns:
```
File "C:\Caelus\cpl\caelus\utils\struct.py", line 157, in load_yaml
  with open(filename, 'r') as fn:
IOError: [Error 13] Permission denied: c:\\Users\\cnsidero\\Documents\\caelus"
```
where CAELUSRC=C:\Users\cnsidero\Documents\caelus. I tried various location in which my user account has rwx permissions but same issue.

- Adding MSMPI support was dropped for 8.04
